### PR TITLE
Document needed sebooleans for httpd/zabbix-web

### PR DESCRIPTION
**Pull request by Fabian1976**
The web.pp missed these sebooleans:

- httpd_can_network_connect
- httpd_can_network_connect_db

As mentioned in issue #477

### CODE REVIEW

**Comment by bastelfreak** [comment]
Hi @Fabian1976, thanks for this PR. I'm not sure if we should include those sebooleans in the module. They are apache specific, not zabbix specific. It is possible, that others manage this seboolean already, but outside of the module. Maybe it is better to handle this in a profile and add a note to the readme?

**Comment by ekohl** [comment]
I wonder if ```ensure_resource``` should be used to handle possible duplicate definitions. This is always hard and IMHO an unsolved problem.

**Comment by Fabian1976** [comment]
You could also fix this in a profile. But if you include zabbix::web (which needs apache), people assume that it sets the appropiate SEbooleans (it sets 1, why not all that are needed for web to work?)

**Comment by bastelfreak** [comment]
There is a small but important difference. It sets all needed sebooleans that httpd needs to talk to zabbix. It doesn't  set all sebooleans that httpd need. IMO we shouldn't manage those sebooleans in the zabbix module. They should be part of httpd module or a profile.

**Comment by Fabian1976** [comment]
I can live with that. I allready added it to a profile as a work around. But then i will make it permanent. Not a problem. Shall i close my PR?

**Comment by ekohl** [comment]
It would be great if it's at least documented in the README.

**Comment by bastelfreak** [comment]
@Fabian1976 can you remove the current changes and instead document it in the README.md?

**Comment by Fabian1976** [comment]
Yes, not a problem. Will do

**Comment by Fabian1976** [comment]
I updated the README

File: README.md [hunk 1]
```diff
@@ -169,6 +169,16 @@ node 'zabbix.example.com' {
 Everything will be installed on the same server. There is also an possibility to seperate the components, please check the following wiki:
 https://github.com/voxpupuli/puppet-zabbix/wiki/Multi-node-Zabbix-Server-setup
 
+Please note that if you use apache as the frontend (which is the default) and SELinux is enabled, you need to set these SEBooleans (preferably in a profile) to allow apache to connect to the database:
+```ruby
```
Comment (ekohl) [inline README.md]: There's puppet highlighting too

File: README.md [hunk 2]
```diff
@@ -169,6 +169,16 @@ node 'zabbix.example.com' {
...
+if $facts['selinux'] {
+  selboolean { ['httpd_can_network_connect', 'httpd_can_network_connect_db']
```
Comment (ekohl) [inline README.md]: Missing a colon here

Comment (Fabian1976) [reply README.md]: Fixed it

**Pull request merged**